src/libraries/System.Net.Security/src/System/Net/Security/SslCertificateTrust.cs

```diff
@@ -0,0 +1,66 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Versioning;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net.Security
+{
+    public sealed partial class SslCertificateTrust
```

Comment:
Why is this partial?

Reply:
not at the moment. I expected that there will be .PLATFORM.cs as well but it is handled by small `#if` branch for now. updated.